www/data_module/README.md

```diff
@@ -25,3 +25,17 @@ $ gulp
 $ npm install
 $ karma start
 ```
+
+## How to test within buildbot/www/base ?
+
+```
+$ cp dist/* ../base/libs/buildbot-data/dist/
```

Comment:
are you sure this works?

guanlecoja is supposed to erase the dist directory at startup, which will delete your symlink.